### PR TITLE
Fix webcam on safari

### PR DESCRIPTION
**Pull request by hjheath**

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-282-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-282-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://282-pr-onfido-sdk-ui-onfido.surge.sh

File: package.json [hunk 1]
```diff
@@ -100,7 +100,7 @@
     "react-native-listener": "1.0.1",
     "react-phone-number-input": "^0.15.2",
     "react-redux": "4.4.6",
-    "react-webcam-onfido": "0.0.19",
+    "react-webcam-onfido": "0.0.20",
```
Comment (stefaniacardenas) [inline package.json]: It's the first time I see this module. Is it a fork?

Comment (hjheath) [reply package.json]: Yeah. you can see the related changes here: https://github.com/onfido/react-webcam/pull/10

**Pull request merged**